manifests/osfamily/aix.pp

```diff
@@ -11,9 +9,23 @@
     fail('AIX upgrades are only supported on Puppet Enterprise')
   }
 
-  class { 'puppet_agent::prepare::package':
-    package_file_name => $package_file_name,
+  $pe_server_version = pe_build_version()
+
+  $_aix_ver_number = regsubst($::platform_tag,'aix-(\d+\.\d+)-power','\1')
```

Comment:
I feel like if I were learning this codebase later, I'd appreciate a comment here about why this this part is necessary for puppet 4 and 5